Documentation/user-guides/getting-started.md

```diff
@@ -146,14 +150,14 @@ spec:
         app: example-app
     spec:
       containers:
-      - name: example-app 
+      - name: example-app
```

Comment:
The tests failed as this file is templated into this markdown file from `../../example/user-guides/getting-started/example-app-deployment.yaml`. You need to remove the space from there as well.